db/migrate/20160901011120_create_pieces.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePieces < ActiveRecord::Migration
+  def change
+    create_table :pieces do |t|
+      t.integer :piece_id 
+      t.integer :user_id 
+      t.string :color 
+      t.integer :game_id 
```

Comment:
FYI we'll need to use `x_position` `y_position` - hyphens are bad practice in schema design. My bad I missed that in the sheet